src/main/java/de/szut/lf8_project/lf8/project/ProjectController.java

```diff
@@ -160,4 +158,27 @@ public ResponseEntity<HashMap<Long, String>> getAllEmployeesByProjectId(@Request
 
         return new ResponseEntity<>(employeeResponseDTOList, HttpStatus.OK);
     }
+
+    @Operation(summary = "Delete a projects employee")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode =  HTTPCodes.SUCCESSFUL, description = "employee was deleted", content = {
+                    @Content(mediaType = MediaTypes.JSON,
+                            schema = @Schema(implementation = GetProjectDto.class))
+            }),
+            @ApiResponse(responseCode = HTTPCodes.NOT_FOUND, description = "not found",
+                    content = @Content),
+            @ApiResponse(responseCode = HTTPCodes.INTERNAL_SERVER_ERROR, description = "internal server error",
+                    content = @Content)
+    })
+    @DeleteMapping (path = "{projectId}/delete/employee/{employeeId}")
+    public void deleteEmployeeFromProject(@PathVariable Long employeeId, @PathVariable Long projectId) throws IOException {
+        ProjectEntity project = service.readById(projectId);
+        if(project == null)
+            throw new InvalidDataException("Project not found");
+
+        if(project.getEmployees().contains(employeeId))
+            project.getEmployees().remove(employeeId);
+        else
+            throw new InvalidDataException("Employee could not be deleted");
+    }
```

Comment:
Du musst dich entscheiden. Du kannst nicht in der Swagger Doc sagen "Es wird das Projekt als JSON zurückgegeben" und dann einfach nichts zurück geben. Zudem könnte es sinnvoll sein das Projekt mit zurück zu geben, damit der Client direkt die richtigen Daten hat. Zudem würde es dadurch konsistenter werden, da wir das auch so bei anderen Endpunkten gemacht haben